share/migrations/0007_auto_20161122_1810.py

```diff
@@ -26,7 +26,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
```

Comment:
Are there any changes in this migration? Looks like it's just choices in a different order.